packages/loader/driver-utils/src/error.ts

```diff
@@ -3,11 +3,13 @@
  * Licensed under the MIT License.
  */
 
+// eslint-disable-next-line import/no-deprecated
 import { DriverErrorType, IDriverErrorBase } from "@fluidframework/driver-definitions";
 import { IFluidErrorBase, LoggingError } from "@fluidframework/telemetry-utils";
 
 /** Error indicating an API is being used improperly resulting in an invalid operation. */
 export class UsageError extends LoggingError implements IDriverErrorBase, IFluidErrorBase {
+	// eslint-disable-next-line import/no-deprecated
 	readonly errorType = DriverErrorType.usageError;
```

Comment:
Can't we use the new DriverErrorTypes usage error here?

Reply:
Same with the other usages of DriverErrorTypes below

Reply:
Unfortunately there are a number of places where we can't replace the type usages without making the change breaking to our consumers. I updated as many places as I believe are possible in a non-breaking way. Since `IDriverErrorBase` types its `errorType` param as `DriverErrorType`, these usages can't change in this PR. They will change on `next` once I make the follow-up breaking change.